pkg/install/generate.go

```diff
@@ -442,6 +444,7 @@ func GenerateInstallerJob(
 	podSpec := provision.Spec.PodSpec
 	podSpec.NodeSelector = nodeSelector
 	podSpec.Tolerations = tolerations
+	podSpec.ImagePullSecrets = append(podSpec.ImagePullSecrets, imagePullSecrets...)
```

Comment:
Wondering why I put these here instead of in [InstallerPodSpec()](https://github.com/openshift/hive/blob/853824843add16752214c1db8876c5b99bcf3979/pkg/install/generate.go#L154). It's probably because we can't be assured the ClusterProvision is fresh. This makes sense in that light.